Automattic/jetpack
# [RNMobile] Tweak label for "Playback Bar Color" setting

Following the addition of the "Playback Bar Color" setting in https://github.com/Automattic/jetpack/pull/29567, I noticed that I had mistakenly used a different label to the one shown in the designs. This is, therefore, a small PR to address that mistake.

## Proposed changes:

* The setting's label has been changed from `Dynamic Color` to `Playback Bar Color`. In addition, the value shown alongside the label is either `Dynamic` or `Manual`, depending on whether the dynamic setting is enabled or not. 

### Other information:


## Jetpack product discussion

p9ugOq-3mV-p2

## Does this pull request change what data or activity we track or use?

No, it does not.

## Testing instructions:

* Navigate to an existing VideoPress block or create a new one in the app.
* Tap the settings toggle to confirm the label for the setting has changed from `Dynamic Color` to `Playback Bar Color`. 

### Screenshots

| Main sheet  | Subsheet |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/226978946-52332a0e-6f5a-4b91-9b27-b1d76ab42bd8.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/226979139-89aee4d7-fc50-44b4-a079-fc09202aeb8a.png" width="100%"> |
